aldryn_categories/models.py

```diff
@@ -86,4 +87,5 @@ def delete(self, using=None):
         super(TranslatableModel, self).delete()
 
     def __str__(self):
-        return self.safe_translation_getter('name', any_language=True)
+        name = self.safe_translation_getter('name', any_language=True)
+        return escape(name)
```

Comment:
this was the case for me on adding new category (option was rendered incorrectly until I escaped the __str__ method)